platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/model/project/type/ProjectType.java

```diff
@@ -33,7 +33,7 @@
 
     Attribute getAttribute(String name);
 
-    boolean getCanBeMixin();
+    boolean canBeMixin();
```

Comment:
Dto doesn't support `can` like methods, `getCanBeMixin` is also not so good as it should be, so either find another name for this method or implement `can` - like methods in _DTO_ framework :smile: